devops/helm/clamAV/_clamav/configurations/1.0/clamd.conf

```diff
@@ -547,7 +547,7 @@ MaxScanSize 150M
 # Technical design limitations prevent ClamAV from scanning files greater than
 # 2 GB at this time.
 # Default: 100M
-MaxFileSize 30M
+MaxFileSize 4M
```

Comment:
I know this file was already here but should we keep the parent directory name starting with underscore (_clamav)?

Reply:
the _calmav is part of the copy we did from the [source](https://github.com/bcgov/common-hosted-clamav-service)  . So do not want to change it.